src/components/History/HistoryModal.tsx

```diff
@@ -26,48 +44,80 @@ type HistoryModalProps = {
-      ({ boxId, status, txId, summary }) => ({
-        boxId: (
-          <CopyToClipboard text={boxId} onCopy={() => toast.info('Copied')}>
-            <span style={{ cursor: 'pointer' }}>
-              {boxId.slice(0, 16)}...{boxId.slice(48)}
-            </span>
-          </CopyToClipboard>
-        ),
-        status,
-        txId: (
-          <CopyToClipboard text={txId} onCopy={() => toast.info('Copied')}>
-            <span style={{ cursor: 'pointer' }}>
-              {txId.slice(0, 16)}...{txId.slice(48)}
-            </span>
-          </CopyToClipboard>
-        ),
-        // operation: <div>{summary}</div>,
-      }),
-    );
+
+    const formattedOperations = operations.map(({ boxId, status, txId }) => ({
+      boxId: (
+        <CopyToClipboard text={boxId} onCopy={() => toast.info('Copied')}>
+          <span style={{ cursor: 'pointer' }}>{truncate(boxId)}</span>
+        </CopyToClipboard>
+      ),
+      status,
+      txId: (
+        <CopyToClipboard text={txId} onCopy={() => toast.info('Copied')}>
+          <span style={{ cursor: 'pointer' }}>{truncate(txId)}</span>
+        </CopyToClipboard>
+      ),
+      operation: (
+        <Container>
+          <Col>
+            <Tooltip text={'View on Explorer'} type="dark">
+              <Button
+                icon={<FontAwesomeIcon icon={faExternalLinkAlt} />}
+                auto
+                size="small"
+                onClick={() => exploreTx(txId)}
+              />
+            </Tooltip>
+          </Col>
+          {isRefundableOperation(status) && (
+            <>
+              <Spacer x={0.2} />
+              <Col>
+                <Tooltip text={'Refund transaction'} type="dark">
+                  <Button
+                    auto
+                    size="small"
+                    onClick={handleOpen}
+                    icon={<FontAwesomeIcon icon={faUndo} />}
+                  />
+                </Tooltip>
+              </Col>
+              <ConfirmRefundModal
+                txId={txId}
+                open={open}
+                onClose={handleClose}
+              />
+            </>
+          )}
+        </Container>
+      ),
+    }));
     return (
       <Table data={formattedOperations}>
         <Table.Column prop="boxId" label="Box ID" />
         <Table.Column prop="txId" label="TX ID" />
-        {/* <Table.Column prop="operation" label="Operation" /> */}
         <Table.Column prop="status" label="Status" />
-        {/* <Table.Column prop="summary" label="Summary" /> */}
+        <Table.Column prop="operation" />
+        {/*<Table.Column prop="summary" label="Summary" />*/}
       </Table>
     );
   },
 );
 
 export const HistoryModal = (props: HistoryModalProps): JSX.Element => {
   const { open = false, onClose } = props;
-  const [operations, setOperations] = useState<AmmOperation[] | null>(null);
+  const [operations, setOperations] = useState<AmmOperation[] | null>([
```

Comment:
I prefer to use `undefined` instead of `null` for the bottom type(unspecified state option). But that is up to you.